.github/workflows/daily.yml

```diff
@@ -1,32 +0,0 @@
-name: Daily tests
```

Comment:
Why do we remove the tests?

Reply:
these were some notebook tests that I tried to add in one commit but then it was too long so I removed them in the last commit
I just kept the commit to keep it tracked

Reply:
ok